app/timedatectl/Template_App_timedatectl_active.xml

```diff
@@ -0,0 +1,175 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<zabbix_export>
+    <version>3.0</version>
+    <date>2017-02-05T01:53:46Z</date>
+    <groups>
+        <group>
+            <name>App templates</name>
+        </group>
+    </groups>
+    <templates>
+        <template>
+            <template>Template App timedatectl active</template>
+            <name>Template App timedatectl active</name>
+            <description/>
+            <groups>
+                <group>
+                    <name>App templates</name>
+                </group>
+            </groups>
+            <applications>
+                <application>
+                    <name>timedatectl</name>
+                </application>
+            </applications>
+            <items>
+                <item>
+                    <name>NTP enabled</name>
+                    <type>7</type>
+                    <snmp_community/>
+                    <multiplier>0</multiplier>
+                    <snmp_oid/>
+                    <key>rabe.timedatectl.ntp.enabled</key>
+                    <delay>3600</delay>
```

Comment:
Do we really want to wait for up to 1 hour to figure out, that NTP hasn't been enabled? I would suggest do lower this to the RaBe default of 300 seconds and lower the history and or trends accordingly (as this will only be used for triggers to fire and not for long term statistic values.

Reply:
The synchronized check will let let you know during provisioning.